### PR TITLE
Smoke test added

### PR DESCRIPTION
**Pull request by SDETEMRE**
Smoke test added

### CODE REVIEW

File: src/test/java/com/symund/runners/CukesRunner.java [hunk 1]
```diff
@@ -14,7 +14,7 @@
         features = "src/test/resources/features",
         glue = "com/symund/step_definitions",
         dryRun = false,
-        tags = "@wit"
```
Comment (HikmetMelek) [inline src/test/java/com/symund/runners/CukesRunner.java:17]: delete this

File: src/test/java/com/symund/pages/FilesPage_Melek.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+import com.symund.utilities.BrowserUtils;
+import com.symund.utilities.Driver;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+public class FilesPage_Melek extends BasePage{
```
Comment (HikmetMelek) [inline src/test/java/com/symund/pages/FilesPage_Melek.java:11]: have to change

File: src/test/java/com/symund/step_definitions/Files_Emre_StepDefs.java [hunk 3]
```diff
@@ -1,37 +1,72 @@
 package com.symund.step_definitions;
 
-import com.symund.pages.FilesPage;
+import com.symund.pages.FilesPage_Melek;
```
Comment (HikmetMelek) [inline src/test/java/com/symund/step_definitions/Files_Emre_StepDefs.java:3]: this was changed

**Review by HikmetMelek — COMMENTED** [review]
need some changes

File: src/test/java/com/symund/runners/CukesRunner.java (hunk 1 above)
Comment (SDETEMRE) [reply src/test/java/com/symund/runners/CukesRunner.java:17]: done

File: src/test/java/com/symund/pages/FilesPage_Melek.java (hunk 2 above)
Comment (SDETEMRE) [reply src/test/java/com/symund/pages/FilesPage_Melek.java:11]: done

File: src/test/java/com/symund/step_definitions/Files_Emre_StepDefs.java (hunk 3 above)
Comment (SDETEMRE) [reply src/test/java/com/symund/step_definitions/Files_Emre_StepDefs.java:3]: done

**Comment by SDETEMRE** [comment]
I changed your requests

**Comment by HikmetMelek** [comment]
thanks. ellerine sağlık

**Pull request closed without merging**